src/main/java/idk/plugin/npc/Loader.java

```diff
@@ -15,6 +16,12 @@
     @Override
     public void onEnable() {
         NPC.registerEntity();
+        PluginManager pluginManager = Server.getInstance().getPluginManager();
+        if(pluginManager.getPlugin("FormAPI") == null) {
```

Comment:
You can't expect it to be always loaded before this plugin. You should list it in the dependencies.